package.json

```diff
@@ -28,7 +28,7 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "lerna": "^3.22.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.1",
+    "prettier": "^2.8.3",
```

Comment:
Did anything in particular prompt this change?

Reply:
reviewing `yarn outdated` and being eager adopting in formatting tool changes to reduce churn (assuming over time we have more code that's likely to be affected)